app/assets/stylesheets/components/_theme.scss

```diff
@@ -14,31 +14,28 @@ $Theme-fontSizes: (
 }
 
 $Theme-blue: #2E86AB;
-$Theme-gray: #E2E1E2;
 $Theme-greenSuccess: #566612;
 $Theme-lightGreenSuccess: #E6F0C0;
 $Theme-lightRedError: #FCE3E4;
 $Theme-redDanger: #BF283A;
 $Theme-redError: #981215;
 $Theme-white: #FFFFFF;
+$Theme-orange: #FF623F;
 
 $Theme-color: (
   background: (
-    body: $Theme-white,
+    default: $Theme-white,
```

Comment:
It was mainly for two reasons. First, as you said before, these names have a strong connection to the HTML (header, body and footer). The second reason was because it was because the values were the same, `$Theme-white` for all of them. Maybe part of the problem was caused because there are a color palette well defined.